src/app/phase2/new-team-respond/new-team-response.component.ts

```diff
@@ -52,6 +57,24 @@ export class NewTeamResponseComponent implements OnInit {
       this.duplicateOf.updateValueAndValidity();
       this.responseTag.updateValueAndValidity();
     });
+
+    if (this.issue.severity === '') {
+      this.selectedSeverityColor = 'ffffff';
+    } else {
+      this.selectedSeverityColor = this.issue.severityColor;
+    }
+
+    if (this.issue.type === '') {
+      this.selectedTypeColor = 'ffffff';
+    } else {
+      this.selectedTypeColor = this.issue.typeColor;
+    }
+
+    if (this.issue.responseTag === '') {
+      this.selectedResponseColor = 'ffffff';
+    } else {
+      this.selectedResponseColor = this.issue.responseColor;
+    }
```

Comment:
Use the `... ? ... : ...`?

Reply:
Also, the code seems to be repetitive. Abstract out?

Reply:
I was thinking whether the 3 issue's attribute (typeColor, severityColor, responseColor) is necessary. Since the colors for the respective labels already exists in `LabelService`, we can have a mapping of different label type to their colors. If no mapping is found, can just simply return `white color`.

Reply:
I'm not sure if this might cause performance issue, as for every issue in the table, I have to look through the list of labels to find the correct color to apply to the issue.

Reply:
How about using a HashMap instead of a List?

Reply:
I have made changes to the code, all label will now refer to the label map in the label service to get the color